### PR TITLE
[ODS-4028] Nugets for Sample Extensions  

### PR DESCRIPTION
**Pull request by semalaiappan**

### CODE REVIEW

File: Application/EdFi.Ods.Extensions.Sample/EdFi.Ods.Extensions.Sample.nuspec [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    <owners>Ed-Fi Alliance</owners>
+    <projectUrl>https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation</projectUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>EdFi.Ods.Extensions.Sample</description>
+    <releaseNotes></releaseNotes>
+    <license type="file">LICENSE.txt</license>
+  </metadata>
+  <files>
+    <file src="../../LICENSE.txt" />
+		<file src="Artifacts/**/*.sql" target="."/>
```
Comment (tbmeador-dlp) [inline Application/EdFi.Ods.Extensions.Sample/EdFi.Ods.Extensions.Sample.nuspec:16]:
```suggestion
    <file src="Artifacts/**/*.sql" target="."/>
```

File: Application/EdFi.Ods.Extensions.Homograph/EdFi.Ods.Extensions.Homograph.nuspec [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    <owners>Ed-Fi Alliance</owners>
+    <projectUrl>https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation</projectUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>EdFi.Ods.Extensions.Homograph</description>
+    <releaseNotes></releaseNotes>
+    <license type="file">LICENSE.txt</license>
+  </metadata>
+  <files>
+    <file src="../../LICENSE.txt" />
+    <file src="Artifacts/**/*.sql" target="."/>
```
Comment (tbmeador-dlp) [inline Application/EdFi.Ods.Extensions.Homograph/EdFi.Ods.Extensions.Homograph.nuspec:16]: we need to add a line here to include the binary files for both packages

Comment (semalaiappan) [reply Application/EdFi.Ods.Extensions.Homograph/EdFi.Ods.Extensions.Homograph.nuspec:16]: I added it . Thank you !

File: Application/EdFi.Ods.Extensions.Sample/EdFi.Ods.Extensions.Sample.nuspec (hunk 1 above)
Comment (semalaiappan) [reply Application/EdFi.Ods.Extensions.Sample/EdFi.Ods.Extensions.Sample.nuspec]: I fixed it . Thanks

**Pull request merged**